partner_tlr_update/models/res_partner.py

```diff
@@ -213,7 +221,12 @@ def update_contact_person(self, xml, parent_id, address_id):
                 })
 
     @api.multi
-    def update_subsidiary(self, xml):
+    def update_subsidiary(self, xml, contract_nrs):
+        contract_id = xml.find(".//ns64:avtalId",
+                                 namespaces={'ns64': 'http://arbetsformedlingen.se/datatyp/tjansteleverantor/utforandeverksamhet/v15'})
+        if contract_id is None or not contract_id.text in contract_nrs:
```

Comment:
Could be replaced with: If not (contract_id and contract_id.text in contract_nrs):